content/md/en/docs/reference/how-to-guides/offchain-workers/offchain-transactions.md

```diff
@@ -298,9 +298,12 @@ Now, your pallet is ready to send signed transactions on-chain from offchain wor
 
 By default, all unsigned transactions are rejected in Substrate.
 To enable Substrate to accept certain unsigned transactions, you must implement the `ValidateUnsigned` trait for the pallet.
-Although you must implement the `ValidateUnsigned` trait to send unsigned transactions, this check doesn't guarantee that a transaction submitted by the offchain worker is a valid transaction or that the transaction hasn't been tampered with.
+Although you must implement the `ValidateUnsigned` trait to send unsigned transactions, this check doesn't guarantee that a transaction was submitted by the offchain worker, that the transaction is a valid transaction, or that the transaction hasn't been tampered with.
 Unsigned transactions always represent a potential attack vector that a malicious user could exploit and offchain workers can't be assumed to be a reliable source without additional safeguards.
 
+You should never assume that a transaction was submitted by an offchain worker. 
+You should always verify transactions—signed or unsigned—coming into the blockchain are from a legitimate source and haven't been tampered with.
```

Comment:
I'd remove this line because it is impossible to verify the source of an unsigned transaction.
```suggestion
```